File: src/main/java/com/hedvig/claims/serviceIntegration/memberService/dto/Member.java
package com.hedvig.claims.serviceIntegration.memberService.dto;

import java.time.LocalDate;
import lombok.Value;

@Value
public class Member {

  private String memberId;

  private String firstName;
  private String lastName;

  private LocalDate birthDate;

  private String street;
  private String city;
  private String zipCode;
  private String country;
}

File: src/main/java/com/hedvig/claims/serviceIntegration/memberService/dto/Member.kt
package com.hedvig.claims.serviceIntegration.memberService.dto

import java.time.LocalDate

data class Member(
    val memberId: String,
    val firstName: String? = null,
    val lastName: String? = null,
    val birthDate: LocalDate? = null,
    val street: String? = null,
    val city: String? = null,
    val zipCode: String? = null,
    val country: String? = null
)
